webhook_to_fedora_messaging/endpoints/user.py

```diff
@@ -0,0 +1,58 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.user import User
```

Comment:
Ref. https://github.com/fedora-infra/webhook-to-fedora-messaging/pull/24#discussion_r1667901072